lib/terraforming/resource/iam_policy.rb

```diff
@@ -44,7 +44,7 @@ def tfstate
       private
 
       def iam_policies
-        @client.list_policies(scope: "Local").collect {|r| r.policies}.flatten
+        @client.list_policies(scope: "Local").map(&:policies).flatten
```

Comment:
__[rubocop]__
* Prefer single-quoted strings when you don't need string interpolation or special symbols. [:ref](https://github.com/bbatsov/ruby-style-guide#consistent-string-literals)